config.in

```diff
@@ -38,7 +38,7 @@ output * bg @datadir@/backgrounds/sway/Sway_Wallpaper_Blue_1920x1080.png fill
 # exec swayidle -w \
 #          timeout 300 'swaylock -f -c 000000' \
 #          timeout 600 'swaymsg "output * dpms off"' \
-#               resume 'swaymsg "output * dpms on"' \
+#          after-resume 'swaymsg "output * dpms on"' \
```

Comment:
I think the current example is correct. `resume` (execute an action when the user is active again) is not to be confused with `after-resume` (which executes an action after the machine wakes up from suspend).